### PR TITLE
ISPN-6533: not possible to change JDBC store configuration in console.

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.jboss.org/browse/ISPN-6533

### CODE REVIEW

File: src/main/webapp/components/api/cache-creation-controller.service.js [hunk 1]
```diff
@@ -198,13 +209,34 @@ angular.module('managementConsole.api')
...
 
+      CacheCreationControllerClient.prototype.updateJDBCStore = function(steps, address, configuration) {
+        var jdbcType = configuration['jdbc-type'];
+        if (utils.isNullOrUndefined(jdbcType) || jdbcType.length < 1 || utils.isNullOrUndefined(configuration[jdbcType])) {
+          return;
+        }
+
+        // Add step to create/update JDBC store
+        var objectKey = jdbcType.toUpperCase().replace(/-/g, '_');
+        debugger;
```
Comment (vblagoje) [inline src/main/webapp/components/api/cache-creation-controller.service.js]: @ryanemerson what's debugger for?

**Comment by vblagoje** [comment]
@ryanemerson after user enters table values the fields remain permanently dirty upon subsequent loads (their text field background is yellow instead of white). Everything else seems fine. Please see if this PR works well with https://github.com/infinispan/infinispan-management-console/pull/101 and if so integrate both! And please create PRs for downstream

File: src/main/webapp/components/api/cache-creation-controller.service.js (hunk 1 above)
Comment (ryanemerson) [inline src/main/webapp/components/api/cache-creation-controller.service.js]: @vblagoje good spot, that definitely should not be here. It's a bit of a workaround for webstorm's debugger.

**Comment by ryanemerson** [comment]
Integrated with 7dc4866e05ceb77a4adaf779ca2c0e0bf8f37a3b

**Pull request closed without merging**